tsconfig.json

```diff
@@ -35,7 +35,7 @@
     "alwaysStrict": true /* Parse in strict mode and emit "use strict" for each source file. */,
 
     /* Additional Checks */
-    "noUnusedLocals": true /* Report errors on unused locals. */,
+    // "noUnusedLocals": true /* Report errors on unused locals. */,
```

Comment:
This is done because of the generated lexer code. It gives the following errors otherwise:
```
    src/antlr/SqlBaseLexer.ts:8:1 - error TS6133: 'NotNull' is declared but its value is never read.

    8 import { NotNull } from "antlr4ts/Decorators";
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    src/antlr/SqlBaseLexer.ts:9:1 - error TS6133: 'Override' is declared but its value is never read.

    9 import { Override } from "antlr4ts/Decorators";
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    src/antlr/SqlBaseLexer.ts:1426:27 - error TS6133: '_serializedATNSegments' is declared but its value is never read.

    1426   private static readonly _serializedATNSegments: number = 5;
                                   ~~~~~~~~~~~~~~~~~~~~~~

```

If there is other solution - would be happy to apply.